ubuntudesign/www.ubuntu.com
# kill out of IA and dead pages, adding redirects

## Done

Deleted the following pages that were either out of the IA or not used and created redirects

- https://www.ubuntu.com/2013/07/edge-founders/
- https://www.ubuntu.com/2013/07/press-pack-ubuntu-edge/
- https://www.ubuntu.com/desktop/take-the-tour-gallery/
- https://www.ubuntu.com/desktop/take-the-tour/
- https://www.ubuntu.com/info/testing/
- https://www.ubuntu.com/index_asus
- https://www.ubuntu.com/index_dell
- https://www.ubuntu.com/index_dell_latam
- https://www.ubuntu.com/index_ebuyer
- https://www.ubuntu.com/index_in
- https://www.ubuntu.com/index_roadshow

## QA

  * see that http://127.0.0.1:8001/2013/07/edge-founders/ redirects to http://insights.ubuntu.com/2013/08/22/ubuntu-edge-thank-you/
 * see that http://127.0.0.1:8001/2013/07/press-pack-ubuntu-edge/ redirects to https://insights.ubuntu.com/press-centre/
 * see that http://127.0.0.1:8001/desktop/take-the-tour-gallery/ redirects to http://tour.ubuntu.com/
 * see that http://127.0.0.1:8001/desktop/take-the-tour/ redirects to http://tour.ubuntu.com/
 * see that http://127.0.0.1:8001/index_in/ redirects to /?: 
 * see that http://127.0.0.1:8001/index_dell/ redirects to http://partners.ubuntu.com/dell
 * see that http://127.0.0.1:8001/index_dell_latam/ redirects to http://partners.ubuntu.com/dell
 * see that http://127.0.0.1:8001/index_ebuyer/ redirects to /
 * see that http://127.0.0.1:8001/index_roadshow/ redirects to /
 * see that http://127.0.0.1:8001/info/testing/ redirects to https://wiki.ubuntu.com/QATeam
